stable_baselines3/common/env_checker.py

```diff
@@ -129,8 +129,8 @@ def _check_goal_env_obs(obs: dict, observation_space: spaces.Dict, method_name:
     (previously known as GoalEnv in gym) contains three elements,
     namely `observation`, `desired_goal`, and `achieved_goal`.
     """
-    assert len(observation_space.spaces) == 3, (
-        "A goal conditioned env must contain 3 observation keys: `observation`, `desired_goal`, and `achieved_goal`."
+    assert len(observation_space.spaces) >= 3, (
```

Comment:
Just curious, was this change caused by the shift to gymnasium?

Reply:
It was because of the HER replay buffer rewrite